__tests__/reactTags.test.js

```diff
@@ -639,3 +641,37 @@ test('should allow duplicate tags when allowUnique is false', () => {
     },
   ]);
 });
+
+test('should display input field and tags inline when inputFieldPosition is passed as inline', () => {
```

Comment:
can you wrap these three in a describe block with title `Test inputFieldPosition`